### ISSUE TITLE
EasyWoW

### ISSUE THREAD

**Issue by sanctum32** [Abd-ulRahman/WoW#1]
Hello, just wanted to note that http://github.com/easywow is active again and well its going in correct way as it should. If you will find something useful, can use it if want. Most annoying issues were fixed https://github.com/EasyWoW/wotlk-core/blob/master/CHANGELOG.

**Comment by Abd-ulRahman** [comment]
Appreciate you're telling me that 
I want you to know the problem with arcemu developers that every one want to be a leader, they do not respect each other opinion and they do not want to work as one team
that what is making it down .

**Comment by sanctum32** [comment]
yea, basically i do not want to be higher than others or similar, i know about ascemu project for example but i have different ideas where in other projects hardly can be realized. basically no one isn't listening to me in arcemu communities. For example creature texts or multidimensional data or new ways how increase scripting/performance...

I have idea to make code well understandable and easy to inspect, read it and by all I also want to learn more and practice my programming skills, that project EasyWoW is good chance to improve or at all skill up my skills, learn more about game. 

I tried to find developers but as you see they alternatively creates own super projects and... they fail. If you have idea, tell me about it and explain it correctly and will try to change/improve/add something. Bigger tasks requires more time, smaller - can be done instantly (simple).

basically i'm working alone but everyday i'm getting more and more working systems than at first commit. About old EasyWoW, I personally do not offer to use older version (created on past year), it contains some seriously bad changes made by random devs... Current EasyWoW already contains that features and even more than older EasyWoW. 

Personally sometimes its hard to force to not stop working but I wont stop working on this core this time till I will have posibility to do that. Past time I had no suitible hardware for development, at this time things are different ;)

**Comment by Abd-ulRahman** [comment]
I hope I did get You right.
Wishing you All the best.
And Good Luck.
